app/scripts/lib/util.js

```diff
@@ -169,6 +192,8 @@ export {
   hexToBn,
   BnMultiplyByFraction,
   checkForError,
+  checkForErrorAndLog,
+  checkForErrorAndWarn,
```

Comment:
We're adding both `checkForErrorAndLog` and `checkForErrorAndWarn` here because we found that throwing errors in the content script may cause the dapp to break, so we should only warn of errors for content script(s). We'll continue logging errors per usual in the background. 

---

[According to docs](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/lastError), we're expected to check the lastError in the runtime API callbacks